### DISCUSSION TITLE
Send song progress or total song duration

### DISCUSSION THREAD

**Discussion by Xim1994** [pschatzmann/ESP32-A2DP#506] [Q&A] [answered]
Hello, 

Is it possible to send the current song progress or total song duration and get it on the metadata callback? For example on an android phone using spotify.

Thanks

**Comment by pschatzmann** [comment, marked as answer]
Maybe you can do something with ESP_AVRC_MD_ATTR_PLAYING_TIME if spotify provides this information....

**Reply by Xim1994** [reply]
I assume there isn't any example out there using it?

**Comment by pschatzmann** [comment]
Did you read the [documentation](https://github.com/pschatzmann/ESP32-A2DP?tab=readme-ov-file#support-for-metadata) ?
